web/handlers/admin/invites.go

```diff
@@ -73,8 +73,6 @@ func (h invitesHandler) create(w http.ResponseWriter, req *http.Request) (interf
 	}
 
 	facadeURL := h.urlTo(router.CompleteInviteFacade, "token", token)
-	facadeURL.Host = h.domainName
-	facadeURL.Scheme = "https"
```

Comment:
What happened here? :)

Reply:
`urlTo` was changed a while ago to set this itself. This here went un-noticd in the refactor and is obsolete now.

https://github.com/ssb-ngi-pointer/go-ssb-room/blob/385b98a3a11a4316e1403e7c62ae7ec91a3f78aa/web/utils.go#L99-L104

Reply:
Cool cool